catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/Historian.java

```diff
@@ -332,6 +332,7 @@ private Filter forIds(List<String> ids) {
                 .filter(ci -> ci.getQualifier() == null || ci.getQualifier()
                         .equals(""))
                 .map(ContentItem::getMetacard)
+                .filter(Objects::nonNull)
```

Comment:
I need this as this was added to fix a bug that a test caught. I'll fix the collision when it happens.